source/_patterns/01-molecules/layouts/grids.mustache

```diff
@@ -0,0 +1,98 @@
+<div class="grids">
+  <div class="pad">
+  <ul class="bullet-list">
+    <li><a href="#g1">One Column</a></li>
+    <li><a href="#g2">Two Column</a></li>
+    <li><a href="#g3">Three Column</a></li>
+    <li><a href="#g4">Four Column</a></li>
+  </ul>
+</div>
+
+<div class="pad">
```

Comment:
Using the `.pad` classes to create spacing is totally fine, but we have [this list of helper spacing classes](https://github.com/adventistchurch/alps/blob/master/source/css/_modifier.spacing.scss#L126) to help create uniform spacing without having to add classes to each element. To use it, just wrap a section of direct child elements that you want to have equal distance apart.

```html
<div class="spacing">
  <div></div> <!-- no effect (doesn't create space for the immediate descendant) -->
  <div></div> <!-- spaced -->
  <div>
    <div></div> <!-- no effect -->
  </div> <!-- spaced -->
  <div></div> <!-- spaced -->
<div>
```